src/controllers/users.controllers.js

```diff
@@ -0,0 +1,73 @@
+'use strict';
+
+const { ApiError } = require('../errorHandler/errorHandler');
+const { UserService } = require('../services/users.service');
+
+const getAllUsersController = async(req, res) => {
+  const userService = new UserService();
```

Comment:
Use [a singleton pattern](https://www.freecodecamp.org/news/singleton-design-pattern-with-javascript/) if you want to work with classes.
Create `UserService` once instead of re-creating it every time